specification/extendedlocation/resource-manager/readme.go.md

```diff
@@ -2,26 +2,19 @@
 
 These settings apply only when `--go` is specified on the command line.
 
-```yaml $(go) && !$(track2)
+```yaml $(go)
 go:
   license-header: MICROSOFT_MIT_NO_VERSION
   clear-output-folder: true
 ```
 
-``` yaml $(go) && $(track2)
-license-header: MICROSOFT_MIT_NO_VERSION
-module-name: sdk/resourcemanager/extendedlocation/armextendedlocation
-module: github.com/Azure/azure-sdk-for-go/$(module-name)
-output-folder: $(go-sdk-folder)/$(module-name)
-azure-arm: true
-```
-
```

Comment:
Please revert this part. This disables the generation for Go Track 2 SDK which is mandatory now for every RP that supports Go.

Reply:
This is a good catch, I reverted the unnecessary changes made at our local repo.